packages/mock-service/src/routes/tweets.ts

```diff
@@ -0,0 +1,12 @@
+import express from "express";
+import {
+  GetTweetsController,
+  PostTweetController,
+} from "../controllers/tweets";
+
+const router = express.Router();
+
+router.get("/tweets", GetTweetsController);
+router.post("/tweets", PostTweetController);
+
+export default router;
```

Comment:
Following my previous comment, this module feels more like a controller whereas `GetTweetsController` for instance could be better renamed to a route handler.